apps/squareone/src/pages/api/sentry-example-api.js

```diff
@@ -0,0 +1,5 @@
+// A faulty API route to test Sentry's error monitoring
+export default function handler(_req, res) {
+  throw new Error('Sentry Example API Route Error');
+  res.status(200).json({ name: 'John Doe' });
+}
```

Comment:
We can ditch these example files, but it could be nice to have a page we can use to get a quick check on if the Sentry integration is working:

https://data-dev.lsst.cloud/sentry-example-page